Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -249,7 +251,7 @@ public override void Analyze(AnalyzeContext context)
                                 validatorMessage));
                     }
 
-                    level = FailureLevel.Error;
+                    level = FailureLevel.Note;
```

Comment:
>Note [](start = 41, length = 4)

This isn't right, 'Error' is correct. We always raise visibility on catastrophic errors. Users will then tell us about them so that we can fix. With your change, this bad engine problem could remain hidden for a long time.